ilxutils/ilxutils/api_classes_wrappe.py

```diff
@@ -0,0 +1,6 @@
+import json
```

Comment:
@tmsincomb Is this file used anywhere? It looks like the filename is misspelled.